src/main/java/pages/SettingCompany/GeneralSetting.java

```diff
@@ -0,0 +1,230 @@
+
+    @FindBy(xpath = "(//div//div[@class='input-group ']//input[@class='form-control '])[1]")
+    private WebElement cnpInput;
+
+    @FindBy(xpath = "(//div[@class=\"input-group \"]//input[@class=\"form-control \"])[2]")
+    private WebElement firstNameInput;
+
+    @FindBy(xpath = "(//div[@class=\"input-group \"]//input[@class=\"form-control \"])[3]")
+    private WebElement lastNameInput;
+
+    @FindBy (xpath = "//div[@class=\"input-group with-text\"]//input[@class=\"form-control\"]")
+    private WebElement registrationDate;
+
+    @FindBy(xpath = "(//span[contains(@role,'combobox')])[3]")
+    private WebElement countryOption;
+
+    @FindBy(xpath = "//div[contains(@class,'row')]//div[3]//div[1]//div[2]")
+    private WebElement countrySaved;
+
+    @FindBy(xpath = "(//span[@role='combobox'])[4]")
+    private WebElement countyOption;
+
+    @FindBy(xpath = "//input[@id='input-city']")
+    private WebElement cityInput;
+
+    @FindBy(xpath = "(//div//div[@class=\"input-group \"]//input[@class=\"form-control \"])[5]")
+    private WebElement addressInput;
+
+    @FindBy(xpath = "(//div//div[@class=\"input-group \"]//input[@class=\"form-control \"])[5]")
+    private WebElement contactPersonInput;
+
+    @FindBy(xpath = "(//div//div[@class=\"input-group \"]//input[@class=\"form-control \"])[6]")
+    private WebElement websiteInput;
+
+    @FindBy(xpath = "(//div//div[@class=\"input-group \"]//input[@class=\"form-control \"])[6]")
+    private WebElement emailInput;
+
+    @FindBy(xpath = "(//div//div[@class=\"input-group \"]//input[@class=\"form-control \"])[7]")
+    private WebElement phoneInput;
+
+    @FindBy(xpath = "(//div[contains(@class,'col text-truncate')])[4]")
+    private WebElement phoneSaved;
+
+    @FindBy(xpath = "//button[contains(@class,'btn btn-primary')]")
+    private WebElement saveButton;
+
+    @FindBy(xpath = "(//button[contains(@aria-label,'Close')])[2]")
+    private WebElement closeButtonX;
+
+    // alert messages
+    @FindBy(xpath = "//label[contains(text(),'CUI/CIF trebuie să fie un șir de caractere.')]")
+    private WebElement cuiAlertMessage;
+
+    @FindBy(xpath = "//label[normalize-space()='Campul 'Tip entitate' este obligatoriu.']")
+    private WebElement entityTypeAlertMessage;
+
+    @FindBy(xpath = "//label[normalize-space()='Campul 'Numele firmei' este obligatoriu.']")
+    private WebElement companyNameAlertMessage;
+
+    @FindBy(xpath = "//label[contains(text(),'Campul \"Județ\" este obligatoriu.')]")
+    private WebElement countyAlertMessage;
+
+    @FindBy(xpath = "//label[normalize-space()='Campul \"Localitate\" este obligatoriu.']")
+    private WebElement cityAlertMessage;
+
+    @FindBy(xpath = "//label[contains(text(),'Campul \"Adresă\" este obligatoriu.')]")
+    private WebElement addressAlertMessage;
+
+    @FindBy(xpath = "//div[@class='toast toast-success']")
+    public WebElement successMessage;
+
+    public void getEntityTypeOption(String entityType){
```

Comment:
in loc de acest switch ai fi putut folosi aceea metoda cu List<WebElement> creata in BaseTest

Reply:
ok o sa tin cont de acest luclu  multumesc